docs/pages/button.md

```diff
@@ -40,8 +40,10 @@ Additional classes can be added to your button to change its size and shape.
 <a class="button large" href="#">So Large</a>
 <a class="button expanded" href="#">Such Expand</a>
 <a class="button small expanded" href="#">Wow, Small Expand</a>
+<button class="button test-button">Test button</button>
```

Comment:
hey Andy! learning to do a bug fix. yes that and the example thing below were left over from testing but not meant to be pushed.